README.md

```diff
@@ -29,24 +21,25 @@ To get started with debugging, copy the file to `.vscode/launch.json`, or copy i
 
 Now you can press `CTRL+SHIFT+D` (Windows/Linux) or `SHIFT+CMD+D` (Mac) to open the `Run` tab and execute the `Debug: Serverless Offline` configuration.
 
-The configuration will spawn a `yarn offline` process, so you can configure your execution directly from `package.json`. It will also add `SLS_DEBUG=*` for more extensive logs and `WEBPACK_MODE=development` to produce source maps.
+The configuration will spawn a `yarn offline` process, so you can configure your execution directly from `package.json`. It will also add `SLS_DEBUG=*` for more extensive logs.
 
 For more information on debugging on VS Code, see:
 https://code.visualstudio.com/docs/editor/debugging
 
-
 ## Testing
 
 Tests are located in the `tests` folder an can be invoked by running `yarn test`. These tests will invoke the defined
 actions in a wrapper, where the response can then be tested.
 
+Tests can also be run locally using: 
+
+`yarn test:offline` and `yarn test:unit`
+
```

Comment:
Keep the testing instructions unchanged for now, since the migration to TypeScript doesn't affect them in any way. We do need to improve the docs, but it needs an explanation about what each of those test commands does and what setup is required, which I think has scope for its own PR!